docs/spec/components/schemas/FormStatus.yaml

```
@@ -0,0 +1,41 @@
+allOf:
+  - $ref: '#/components/schemas/FormStatusKey'
+  - type: object
+    required:
+      - attributes
+    properties:
+      attributes:
+        type: object
+        required:
+          - status
+          - created_at
+          - next_form_at
+          - until_next_form
+        properties:
+          status:
+            type: string
+            enum: [ accepted, processed ]
+            description: |
+              Accepted - the data was saved by the service for further processing
+              Processed - the data is processed and stored
+          created_at:
+            type: integer
+            format: int64
+            example: 1721392530
+            description: Form submission time. Unix UTC time.
```

```suggestion
            description: Form submission time. Unix time.
```
Unix time is always UTC